Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/MotionCommands.cpp

```diff
@@ -1282,19 +1281,17 @@ namespace CommandSystem
 
         // Iterate through all motions and remove them.
         AZStd::string commandString;
-        for (uint32 i = 0; i < numMotions; ++i)
+        for (const EMotionFX::Motion* motion : motions)
         {
-            EMotionFX::Motion* motion = motions[i];
-
             if (motion->GetIsOwnedByRuntime())
             {
                 continue;
             }
 
             // Is the motion part of a motion set?
             bool isUsed = false;
-            const uint32 numMotionSets = EMotionFX::GetMotionManager().GetNumMotionSets();
-            for (uint32 j = 0; j < numMotionSets; ++j)
+            const size_t numMotionSets = EMotionFX::GetMotionManager().GetNumMotionSets();
+            for (size_t j = 0; j < numMotionSets; ++j)
```

Comment:
nit - bit weird it's still using j here.

Reply:
Fixed.